app/scripts/TiledPlot.jsx

```diff
@@ -277,28 +740,6 @@ export class TiledPlot extends React.Component {
       });
     }
 
-    if (this.state.customDialog || this.props.customDialog) {
-      const dialogData = this.state.customDialog || this.props.customDialog;
-      if (dialogData.length > 0) {
-        const componentArray = [];
-        const bodyPropsArray = [];
-        dialogData.forEach((dd) => {
-          componentArray.push(dd.bodyComponent);
-          bodyPropsArray.push(dd.bodyProps);
-        });
-
-        this.props.modal.open(
-          <CustomTrackDialog
-            // biome-ignore lint/correctness/noChildrenProp:
-            children={componentArray}
-            bodyProps={bodyPropsArray}
-            onCancel={this.props.closeCustomDialog}
-            title={dialogData[0].title}
-          />,
-        );
-      }
-    }
```

Comment:
Is this totally gone now? I don't know what this was for but it's not in the resglass version at all.